src/plugins/legacy/iterativeClosestPoint/iterativeClosestPointProcess.cpp

```diff
@@ -184,6 +220,36 @@ int iterativeClosestPointProcess::update()
     return medAbstractProcessLegacy::SUCCESS;
 }
 
+
+void iterativeClosestPointProcess::exportTransformMatrix()
+{
+    if (d->exportMatrixState == 2)
+    {
+        QByteArray matrixStr;
+        matrixStr += "# Transformation matrix from " + d->sourceName + " to " + d->targetName + "\n";
```

Comment:
Because you are adding a comment as the first line your export file is not usable as it in meshManipulation toolbox. You need to modify  importTransform function of this toolbox so it can skip commented lines

Reply:
Problem solved